source/camera/basic/capture_2d.py

```diff
@@ -7,14 +7,14 @@ def _main():
     app = zivid.Application()
     camera = app.connect_camera()
 
-    settings_2d = zivid.Settings2D()
-    settings_2d.iris = 20
-    settings_2d.exposure_time = datetime.timedelta(microseconds=30000)
-    settings_2d.brightness(1.8)
-    settings_2d.gain(2.0)
+    settings_2d = zivid.Settings2D(
+        acquisitions=[
+            zivid.Settings2D.Acquisition(aperture=2.83, exposure_time=datetime.timedelta(microseconds=10000),)
+        ],
+    )
```

Comment:
Similarly here. But nice to be consistent in any case.

Reply:
Done